pages/lists/[listId]/contributors.tsx

```diff
@@ -1,9 +1,147 @@
+import useSWR, { Fetcher } from "swr";
+import { useState } from "react";
+import { GetServerSidePropsContext } from "next";
 import ListPageLayout from "layouts/lists";
+import publicApiFetcher from "lib/utils/public-api-fetcher";
+import ContributorTable from "components/organisms/ContributorsTable/contributors-table";
+import PaginationResults from "components/molecules/PaginationResults/pagination-result";
+import ContributorListTableHeaders from "components/molecules/ContributorListTableHeader/contributor-list-table-header";
+import Pagination from "components/molecules/Pagination/pagination";
+import { fetchApiData } from "helpers/fetchApiData";
 
-const ListContributors = () => {
-  return <div>List Contributors</div>;
+interface ContributorList {
+  data: {
+    user_id: string;
+    created_at: string;
+  }[];
+  meta: Meta;
+}
+
+function convertToContributors(rawContributors: ContributorList["data"] = []) {
+  const contributors = rawContributors
+    ? rawContributors.map((contributor) => {
+        return {
+          author_login: contributor.user_id,
+          updated_at: contributor.created_at,
+        };
+      })
+    : [];
+
+  return contributors;
+}
+
+const useContributorsList = (listId: string) => {
+  const { data, error, mutate } = useSWR<ContributorList>(
+    `lists/${listId}/contributors`,
+    publicApiFetcher as Fetcher<ContributorList, Error>
+  );
+  const contributors = convertToContributors(data?.data);
+
+  return {
+    data: data ? { data: contributors, meta: data.meta } : { data: [], meta: {} },
+    isLoading: !error && !data,
+    isError: !!error,
+    mutate,
+  };
+};
+
+export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
+  const { listId } = ctx.params as { listId: string };
+  const { data, error: contributorListError } = await fetchApiData({
+    context: ctx,
+    path: `lists/${listId}/contributors`,
+  });
+  const contributors = convertToContributors(data?.data);
+
+  const { data: list, error } = await fetchApiData({ context: ctx, path: `lists/${listId}` });
+
+  return {
```

Comment:
Add sentry here if there are errors?

Reply:
I'm still not sure of our flow in the app in terms of error reporting.

Reply:
We install sentry in the `next.config` which sets up error reporting for the app